<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmPersonne</class>
 <widget class="QDialog" name="CIhmPersonne">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuPer : Gestion des personnes sur site</string>
  </property>
  <widget class="QLabel" name="lTitre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion des personnes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Remplissez les champs pour une nouvelle personne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ou choisissez une personne dans la liste.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbListe">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Présence au LSBB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>N° interne</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNoPers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date au format YYYY-MM-DD</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbEffacer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Effacer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../usr/share/apache2/icons/alert.red.gif</normaloff>../../../usr/share/apache2/icons/alert.red.gif</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Exploitant ?</string>
   </property>
   <widget class="QRadioButton" name="rbExploitOui">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OUI</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbExploitNon">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NON</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fIdent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Login :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLogin"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Mot de passe :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePass"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>81</y>
     <width>231</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leNom"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Prénom : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePnom"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Société :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSociete"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Date début :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDateDeb">
        <property name="inputMask">
         <string>0000-00-00; </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Date Fin :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDateFin">
        <property name="inputMask">
         <string>0000-00-00; </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>email : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leEmail"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CIhmPersonne</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CIhmPersonne</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbExploitOui</sender>
   <signal>clicked()</signal>
   <receiver>fIdent</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbExploitNon</sender>
   <signal>clicked()</signal>
   <receiver>fIdent</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
